### PR TITLE
contrib/libmpdclient: new package (2.22)

### PR DESCRIPTION
**Pull request by anixedn**

### CODE REVIEW

File: contrib/libmpdclient/template.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+pkgname = "libmpdclient"
+pkgver = "2.22"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["meson", "ninja", "pkgconf"]
+pkgdesc = "Asynchronous API library for interfacing MPD in the C, C+"
```
Comment (nekopsykose) [inline contrib/libmpdclient/template.py:6]:
```suggestion
pkgdesc = "Asynchronous library for interfacing with MPD"
```

File: contrib/libmpdclient/template.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+maintainer = "avgwst <avgwst@tutanota.de>"
+license = "BSD-2-Clause AND BSD-3-Clause"
+url = "https://musicpd.org/libs/libmpdclient"
+source = (
+    f"https://www.musicpd.org/download/{pkgname}/2/{pkgname}-{pkgver}.tar.xz"
+)
+sha256 = "eac15b82b5ba5ed0648af580221eb74657394f7fe768e966d9e9ebb27435429f"
+# no checks
+options = ["!check"]
```
Comment (nekopsykose) [inline contrib/libmpdclient/template.py:15]: it literally has tests

File: contrib/libmpdclient/template.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+build_style = "meson"
+hostmakedepends = ["meson", "ninja", "pkgconf"]
+pkgdesc = "Asynchronous API library for interfacing MPD in the C, C+"
+maintainer = "avgwst <avgwst@tutanota.de>"
+license = "BSD-2-Clause AND BSD-3-Clause"
+url = "https://musicpd.org/libs/libmpdclient"
+source = (
+    f"https://www.musicpd.org/download/{pkgname}/2/{pkgname}-{pkgver}.tar.xz"
+)
+sha256 = "eac15b82b5ba5ed0648af580221eb74657394f7fe768e966d9e9ebb27435429f"
```
Comment (nekopsykose) [inline contrib/libmpdclient/template.py:13]: you should also check if `hardening = ` tags work (if cfi works, if it builds and symbols are fine with `vis`). same for the other templates

Comment (anixedn) [reply contrib/libmpdclient/template.py:13]: thanks! i disabled vis in this template, it seems that it breaks compilation

File: contrib/libmpdclient/template.py (hunk 2 above)
Comment (nekopsykose) [reply contrib/libmpdclient/template.py]: (you need check-devel and -Dtest=true for it. please actually check this, it takes like 20 seconds..)

File: contrib/libmpdclient/template.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
+pkgname = "libmpdclient"
+pkgver = "2.22"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dtest=true"]
+hostmakedepends = ["meson", "ninja", "pkgconf"]
+checkdepends = ["check-devel"]
```
Comment (nekopsykose) [inline contrib/libmpdclient/template.py:7]:
```suggestion
makedepends = ["check-devel"]
```

File: contrib/libmpdclient/template.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
+pkgname = "libmpdclient"
+pkgver = "2.22"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dtest=true"]
+hostmakedepends = ["meson", "ninja", "pkgconf"]
```
Comment (nekopsykose) [inline contrib/libmpdclient/template.py:6]:
```suggestion
hostmakedepends = ["meson", "pkgconf"]
```

**Pull request merged**